engine/storage/volume/src/org/apache/cloudstack/storage/volume/VolumeServiceImpl.java

```diff
@@ -1237,17 +1201,15 @@ private HostVO getHost(Long zoneId, HypervisorType hypervisorType, boolean compu
         return future;
     }
 
-    @Override
     @DB
-    public boolean destroyVolume(long volumeId) throws ConcurrentOperationException {
+    @Override
+    public void destroyVolume(long volumeId) {
         // mark volume entry in volumes table as destroy state
         VolumeInfo vol = volFactory.getVolume(volumeId);
         vol.stateTransit(Volume.Event.DestroyRequested);
         snapshotMgr.deletePoliciesForVolume(volumeId);
 
         vol.stateTransit(Volume.Event.OperationSucceeded);
-
-        return true;
```

Comment:
@rafaelweingartner, I figured out lately, I guess the change of return type because it always returns true??

Reply:
That is it 👍